lib/legacy_mempool.go

```diff
@@ -2077,23 +2094,6 @@ func ComputeTransactionMetadata(
 			})
 		}
 	}
-	// Check if the transactor is an affected public key. If not, add them.
-	// We skip this for atomic transactions as their transactor is the ZeroPublicKey.
-	if txnMeta.TransactorPublicKeyBase58Check != "" && txn.TxnMeta.GetTxnType() != TxnTypeAtomicTxnsWrapper {
-		transactorPublicKeyFound := false
-		for _, affectedPublicKey := range txnMeta.AffectedPublicKeys {
-			if affectedPublicKey.PublicKeyBase58Check == txnMeta.TransactorPublicKeyBase58Check {
-				transactorPublicKeyFound = true
-				break
-			}
-		}
-		if !transactorPublicKeyFound {
-			txnMeta.AffectedPublicKeys = append(txnMeta.AffectedPublicKeys, &AffectedPublicKey{
-				PublicKeyBase58Check: txnMeta.TransactorPublicKeyBase58Check,
-				Metadata:             "TransactorPublicKeyBase58Check",
-			})
-		}
-	}
```

Comment:
This got duplicated somehow. See lines right above.

Reply:
Weird, not sure how this got duplicated.